src/promptflow-tools/promptflow/tools/embedding.py

```diff
@@ -33,6 +45,13 @@ def embedding(connection: Union[AzureOpenAIConnection, OpenAIConnection], input:
             input=input,
             model=model
         ).data[0].embedding
+    elif is_serverless_connection(connection):
```

Comment:
do we plan to add ut in this pr?

Reply:
Yes, we should. I want to add the ut after I tested in local. However I didn't get an embedding deployment for test now.